ncm-download/src/main/pan/components/download/config.pan

```diff
@@ -33,4 +33,4 @@ prefix '/software/components/${project.artifactId}';
 
 'active' ?= true;
 'dispatch' ?= true;
-
+'head_timeout' ?= 3;
```

Comment:
if this is the default, it's better to set it via the schema. about the value, i have no clue if this is a lot or not . but since there was no default previously, a larger default value to mimc no timeout might be appropriate.